sympy/printing/pycode.py

```diff
@@ -845,7 +845,8 @@ def _print_CodegenArrayElementwiseAdd(self, expr):
     'hermite': 'eval_hermite',
     'laguerre': 'eval_laguerre',
     'assoc_laguerre': 'eval_genlaguerre',
-    'beta': 'beta'
+    'beta': 'beta',
+    'LambertW' : 'lambertw'
```

Comment:
Can you add a comma at the end of this line so it doesn't need to be changed when more things are added?